test/rubygems/test_gem_commands_install_command.rb

```diff
@@ -753,6 +753,20 @@ def test_install_gem_ignore_dependencies_remote
     assert_equal %w[a-2], @cmd.installed_specs.map { |spec| spec.full_name }
   end
 
+  def test_install_gem_ignore_dependencies_remote_platform_local
+    local = Gem::Platform.local
+    spec_fetcher do |fetcher|
+      fetcher.gem 'a', 3
+      fetcher.gem 'a', 3 do |s| s.platform = local end
```

Comment:
I would wrap this in curly brackets instead of `do end` for single lines.

Reply:
Have to add parens?
```ruby
      fetcher.gem 'a', 3
      fetcher.gem('a', 3) { |s| s.platform = local end }
```

Reply:
I can also make it multiline to keep the surrounding parens style

```ruby
      fetcher.gem 'a', 3
      fetcher.gem('a', 3) do |s|
        s.platform = local
      end
```

Reply:
I had a look at other places and the style used in this PR seems the one being most commonly used for this kind of definitions in general. I like other styles better, so I'll try to enable another style on a separate rubocop PR, but this PR can go in as is :+1:

Reply:
#2640 has been merged, so this is good now!